todoCLI/todoCLI.js

```diff
@@ -0,0 +1,106 @@
+const fs = require('fs').promises;
+const readline = require("readline");
+
+const rl = readline.createInterface({
+    input: process.stdin,
+    output: process.stdout,
+    prompt: ">"
+})
+
+let todoList = [];
+const welcome = "Welcome to Todo CLI!\n--------------------";
+const menu = "(v) View • ( n ) New • (cX) Complete • (dX) Delete • (s) Save • (q) Quit";
+const defaultPath = "myTodos.json";
+
+function ask() {
+    console.log(menu);
+    rl.prompt();
+}
+
+
+function todo() {
+    console.log(welcome);
+    ask();
+
+    rl.on("line", (line) => {
+        switch (line.trim().charAt(0)) {
+            case "v":
+                if (todoList.length == 0) {
+                    console.log("List is empty...");
+                } else {
+                    todoList.forEach((item, index) => {
+                        const completedMark = item.completed ? "[✓]" : "[ ]";
+                        console.log(`${index} ${completedMark} ${item.title}`);
+                    });
+                }
+                ask();
+                break;
+            case "n":
+                rl.question("What?\n>", (answer) => {
+                    let newItem = {};
+                    newItem.title = answer;
+                    newItem.completed = false;
+                    todoList.push(newItem);
+                    ask();
+                })
+                break;
+            case "c":
+                const index = parseInt(line.slice(1));
+                if (!index) {
```

Comment:
when index is 0, it's always falsy so a user will never be able to complete/delete an item in index 0